geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/DataFrameFunctions.scala

```diff
@@ -354,6 +354,9 @@ object DataFrameFunctions extends SpatialEncoders {
 
     def st_difference(geom1:Column, geom2:Column): TypedColumn[Any, Geometry] =
       udfToColumn(ST_Difference, relationNames, geom1, geom2)
+
+    def st_transform(geom: Column, fromCRS: Column, toCRS: Column): TypedColumn[Any, Geometry] =
```

Comment:
i'm a little confused... i thought all the st_transform methods and code would need to be defined in the spark-sql module instead of the spark-jts module, since they require the `org.geotools` dependencies, which we don't want to introduce to the jts module. but i see that `ST_LengthSphere` is in the jts module, even though the function `def st_lengthSphere` is defined in the spark-sql module and also uses geotools? @jnh5y can you clarify?
but I think the bottom line is that we don't want to include the changes you made in the geomesa-spark-jts/pom.xml to add geotools to the jts module.